content/docs/visualization/grafana.md

```diff
@@ -25,15 +25,23 @@ By default, Grafana will be listening on
 
 ### Creating a Prometheus data source
 
-To create a Prometheus data source:
+First, we need to get Prometheus running and listening on port 9090. The easiest way to do this is via Docker:
 
-1. Click on the Grafana logo to open the sidebar menu.
-2. Click on "Data Sources" in the sidebar.
-3. Click on "Add New".
+```
+docker run -p 9090:9090 prom/prometheus
+```
+
+Confirm that you can reach the Prometheus web UI in a browser on http://localhost:9090.
+
+To create a Prometheus data source in Grafana:
+
+1. Click on the "cogwheel" in the sidebar to open the Configuration menu.
+2. Click on "Data Sources".
+3. Click on "Add data source".
 4. Select "Prometheus" as the type.
-5. Set the appropriate Prometheus server URL (for example, `http://localhost:9090/`)
-6. Adjust other data source settings as desired (for example, turning the proxy access off).
-7. Click "Add" to save the new data source.
+5. Set the appropriate Prometheus server URL (in this case it's `http://localhost:9090/`)
+6. We need to use `Browser` mode because so that Grafana can connect to Prometheus directly.
```

Comment:
This doesn't seem correct, and proxy mode is generally what is used. I'd leave this as-is.

Reply:
I've changed back  now and used the correct wording as per current Grafana UI, which refers to Access methods.